cli/command/test_helpers.go

```diff
@@ -0,0 +1,37 @@
+package command
+
+import (
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+var TEST_TIMEOUT = time.Minute * 15
+
+func GetCLIContext(t *testing.T, args []string, flags map[string]interface{}) *cli.Context {
+	flagSet := &flag.FlagSet{}
+
+	for key, val := range flags {
+		switch v := val.(type) {
+		case bool:
+			flagSet.Bool(key, v, "")
+		case string:
+			flagSet.String(key, v, "")
+		case []string:
+			slice := cli.StringSlice(v)
+			flagSet.Var(&slice, key, "")
+		case int:
+			flagSet.Int(key, v, "")
+		default:
+			t.Errorf("Cannot generate CLI context: unknown flag type for '%s'", key)
+		}
+	}
+
+	// add default global flags
+	flagSet.String("output", "text", "")
```

Comment:
this should use the consts in the `config` package. I think it's something like:
```
flagSet.String(config.FLAG_TIMEOUT, "15m", "")
```

Reply:
yep good guess ;). 

What's the point of `flagSet.String("output", "text", "")`?

Reply:
`l0 -output=json|text ...`

Reply:
✅